api/RTCPeerConnection.json

```diff
@@ -105,13 +105,22 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/RTCPeerConnection",
           "support": {
             "webview_android": {
-              "version_added": "56"
+              "version_added": "56",
+              "notes": [
+                "Before Chrome 63 the default value for the <code>rtcpMuxPolicy</code> property was <code>&quot;negotiate&quot</code>"
```

Comment:
This might be just me, but I think that "property" might be confused as a property of `RTCPeerConnection` and not the `RTCConfiguration` dictionary. Can this be reworded to be explicit that it's a property of `RTCConfiguration`? Maybe it would be enough to use the "member" terminology from the spec